pyvista/core/filters/data_set.py

```diff
@@ -2807,9 +3299,21 @@ def surface_indices(dataset):
 
         Returns
         -------
-        surf_ind : np.ndarray
+        np.ndarray
             Indices of the surface points.
 
+        Examples
+        --------
+        Return the surface indices of an UnstructuredGrid.
+
+        >>> from pyvista import examples
+        >>> grid = examples.load_hexbeam()
+        >>> ind = grid.surface_indices()
+        >>> ind  # doctest:+SKIP
+        pyvista_ndarray([ 0,  2, 36, 27,  7,  8, 81,  1, 18,  4, 54,
+        ...
+                         65, 66, 67, 68, 69, 70])
```

Comment:
Same remark as above about presentation.

Reply:
I feel like this is fine.  Do you not like the way the array is presented?  Should it be a single line?

Reply:
The leading ellipsis conflicted my mental parser. It looks like a REPL continuation line, but it isn't. It's a throwaway symbol denoting that lines in the output were snipped out.

Perhaps just moving the ellipsis would help?
```
        pyvista_ndarray([ 0,  2, 36, 27,  7,  8, 81,  1, 18,  4, 54,
                         ...
                         65, 66, 67, 68, 69, 70])
```

I guess that's a different kind of ugly :/

Again this is nothing crucial, just something that stood out to me. It's alright if we leave it as-is.

Reply:
What about:
```py
        pyvista_ndarray([ 0,  2, 36, 27,  7,  ..., 81,  1, 18,  4, ])
```

Reply:
Actually:
```
        Access the first 4 elevation scalars.  This is a point-wise
        array containing the "elevation" of each point.

        >>> sphere_elv['Elevation'][:4]  # doctest:+SKIP
        array([-0.5       ,  0.5       , -0.49706897, -0.48831028], dtype=float32)
```

Reply:
Better, and perfect!